### ISSUE TITLE
Kill active session without deleting

### ISSUE THREAD

**Issue by frisedel** [ridvanaltun/guacamole-rest-api-documentation#7]
Is there a "Kill session" call for this api? In the client UI, there is a button doing just that but I can't seem to find any API call for it. I tried "Kill connection" but that is ie the same as "Delete connection", but using patch rather than delete.

**Comment by AustinLeath** [comment]
> Is there a "Kill session" call for this api? In the client UI, there is a button doing just that but I can't seem to find any API call for it. I tried "Kill connection" but that is ie the same as "Delete connection", but using patch rather than delete.

There very well may be a kill session api route, this documentation is out of date, but is the best I have found. I will do some digging with postman and see what i can find out for you this evening.

**Comment by ridvanaltun** [comment]
This document is based on Guacamole version 1.1.0 and the latest version is 1.4.0, differences may possible.

**Comment by frisedel** [comment]
> This document is based on Guacamole version 1.1.0 and the latest version is 1.4.0, differences may possible.

yes, that is true, but this is still one of the better documents for the rest api that I have found :)

**Comment by ridvanaltun** [comment]
As far I understand you want to make an HTTP request that kills sessions like the below button do:

![Screenshot 2022-02-24 002459](https://user-images.githubusercontent.com/10441216/155411142-743ad00c-1f5e-41db-932d-295a47b47157.png)

I have some digging and just found out nothing changed in the Guacamole version 1.4.0 for killing sessions. You can use this endpoint without any modification: https://github.com/ridvanaltun/guacamole-rest-api-documentation/blob/master/docs/CONNECTIONS.md#kill-connections
